promotions/lib/generators/solidus_promotions/install/install_generator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SolidusPromotions
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
```

Comment:
This could be a good place to share some info/doc with people installing the new gem.